daemon/snapshotter/mount_default.go

```diff
@@ -0,0 +1,17 @@
+//go:build !windows
+
+package snapshotter
+
+import (
+	"github.com/containerd/containerd/mount"
+	"github.com/docker/docker/daemon/graphdriver"
```

Comment:
Feels a bit weird to make `daemon/snapshotter` depend on the `daemon/graphdriver`.
How cumbersome would it be to extract the common bits out of the `daemon/graphdriver`?

Reply:
Yeah I wanted to keep this PR small-ish, I can add a commit to move it or in a followup